text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
+* Highlight contributions made by community members (plugins built, code PRs)
+* "Official" landing pages to help community members in their personal website marketing efforts
+* Allow companies interested in building Gatsby sites to find individuals nearby them who are open for Gatsby projects
+
+#### Agency Pages
+* Highlight agencies building websites on Gatsby
+* "Official" landing pages to help agencies in their marketing efforts
+* Allow companies interested in building Gatsby sites to find agencies nearby them
+* Help agencies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+#### Company Pages
+* Display list of companies using Gatsby to demonstrate it's "enterprise-ready" and fulfills whatever use-case they're looking for (e-commerce, authentication, etc). 
+* Help companies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+# Detailed design
+
+## Pages
+
+#### All people / agencies / companies
+
+One template with multiple URLs, each of which apply different filters:
+  * /people/ (all people)
+  * /agencies/ (all agencies)
+  * /companies/ (all companies)
+
+Filter buttons should be at the top and should be easy to toggle between them (select-one). Filters should include:
+  * Individual 
+  * Agency 
+  * Company
+  * Open to new contracts (people & agencies with this field toggled)
+  * Now hiring (agencies & companies with this field toggled)
+
+#### Person / agency / company page
+
+One template with multiple URL patterns, depending on the Creator type:
+  * /companies/[company-name]
+  * /agencies/[agency-name]
+  * /people/[person-name]
+
+
+### Data Structures
+
+Showcase sites ([listed here](https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml)) can have multiple Creators.
+
+**Creators** can either be an Individual, a Digital Agency, or a Company. . 
+
+Creator has the following fields:
+* an id, in slug format (eg mike-allanson)
+* type -- Individual, a Digital Agency, or a Company. We use the word Organization below to refer to either a Digital Agency and Company.
+* a description of them in the third person “Eg, Mike Allanson is a developer living in London"
+* Personal website
+* GH handle (?)
+* Open for Work [boolean]
+* A photo / logo (depending on Individual / Organization type)
+* Org they are affiliated with (if Individual)
```

Comment:
How are we making this work? A company ID? Slug?

Will this be many-to-many (e.g. I could list myself as part of my main company _and_ my side hustle, and my company can list all of its employees)? Or should it be one-to-many only, e.g. you can only specify entities that work with you, rather than listing who you work with?

🚨 Potential for abuse: Nike is a company, and someone adds Nike as an affiliated company, leading them to show up on Nike's page despite there being no relationship.

Reply:
Hmm, after thinking about this a bit more, we might want to add this at some point, but agree that it has the potential for abuse and don't see a huge value add. Will remove.